vmdb/app/models/host_openstack_infra.rb

```diff
@@ -1,2 +1,5 @@
 class HostOpenstackInfra < Host
+  def self.hostname_required?
+    false
+  end
```

Comment:
LOL 2009.  Yeah, I added that if conditional for EC2, but we don't have EC2 hosts anymore, so that's moot.